app/jobs/hyrax/valkyrie_create_derivatives_job_decorator.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+# OVERRIDE to log and report derivative errors to Sentry
+ValkyrieCreateDerivativesJob.class_eval do
```

Comment:
Can we pair up on this? I have an idea of how we could use [a decorator](https://github.com/notch8/playbook/blob/main/dev/ruby/decorators-and-class-eval.md) instead of `#class_eval` that could simplify this override (sneak peak: wrapping `super` in the `begin` block)